templates/field/field--text.html.twig

```diff
@@ -0,0 +1,28 @@
+{% extends "field.html.twig" %}
+{#
+/**
+ * @file
+ * Default theme implementation for a text field.
+ *
+ * A 'clearfix' class is added, because 'text' fields have a 'format' property
```

Comment:
**Thought:** Seems odd to still have clearfix today, but something for future enhancement.

Reply:
This is coming from Drupal so I imagine it will be removed in the future. I've maintained the comments coming from Drupal for reference.